.github/scripts/auto-backport.py

```diff
@@ -192,21 +180,34 @@ def main():
         if not backport_labels:
             print(f'no backport label: {pr.number}')
             continue
-        if args.commits and not with_github_keyword_prefix(repo, pr):
+        if not with_github_keyword_prefix(repo, pr) and 'unlabeled' not in args.github_event:
+            comment = f''':warning:  @{pr.user.login} PR body or PR commits do not contain a Fixes reference to an issue and can not be backported
+            please update commit message or PR body with a valid ref to an issue. Then remove `scylladbbot/backport_error` label to re-trigger the backport process
```

Comment:
since the change has been promoted, force-push to an already-merged PR could be confusing. can we remove "commit message or" from the message?